content/03.methods.md

```diff
@@ -11,93 +11,115 @@ The prompt for the Methods section includes the formatting of equations with ide
 All sections' prompts include these instructions: *"the text grammar is correct, spelling errors are fixed, and the text has a clear sentence structure"*, although these are only shown for abstracts.
 ](images/figure_1.svg "AI-based revision applied on a Manubot manuscript"){#fig:ai_revision width="85%"}
 
-We implemented the AI-based revision infrastructure in Manubot [@doi:10.1371/journal.pcbi.1007128].
-Manubot is a tool for collaborative writing of scientific manuscripts.
-It utilizes version control and a continuous integration workflow to facilitate efficient and transparent collaboration among authors.
+The AI-based revision task is based on a machine learning model trained on a corpus of scientific papers from the same field as the manuscript (Figure {@fig:ai_revision}b).
```

Comment:
Wow! It made this up!

Reply:
Yes, when it revises the Methods section, it usually makes things up (adds text like in here, or equations). I'm guessing that this is probably because of the prompt when it asks to keep technical details. In this case, it added this paragraph on top of the revised one.